translations/strings.yaml

```diff
@@ -634,7 +635,6 @@ subscriptionManagement:
   addFirefoxRelay:
     value: Add Firefox Relay
     comment: Block of text appearing above a link to learn more about the VPN / Relay bundle offering.
-  
```

Comment:
This just happens because my VSCode is set up to apply lints on save... We should probably just have a linter for this on CI honestly.